### DISCUSSION TITLE
Is possible to hide the errors/warnings from Sandpack?

### DISCUSSION THREAD

**Discussion by aaronlin0122** [codesandbox/sandpack#843] [Ideas]
https://neko-nextjs-blog.vercel.app/posts/hello-world#sandpack
![image](https://user-images.githubusercontent.com/47508893/225764032-6a8c83ca-54ac-4b13-bcbd-9f41a133d4fe.png)

**Comment by DeMoorJasper** [comment]
Just had a look at it and we can't fix the tailwind warnings as we don't control those, but will look into getting rid of those babel warnings today.

**Reply by aaronlin0122** [reply]
Thx you!

**Reply by DeMoorJasper** [reply]
The babel warnings have been resolved here: https://github.com/codesandbox/codesandbox-client/commit/274c659223ac8c98b4142a892ecaa1b83396fe2a a new release is being published right now

**Reply by aaronlin0122** [reply]
I found another error it's about CORS:
![image](https://user-images.githubusercontent.com/47508893/225988508-121745bf-0fc8-4376-a337-6226747caee5.png)
